src/lib/containers/SynapseNavDrawer.tsx

```diff
@@ -139,13 +162,14 @@ export const SynapseNavDrawer: React.FunctionComponent<
     setOpen(false)
     setSelectedItem(undefined)
   }
-  const getListItem = (params: MenuItemParams) => {
+  const NavDrawerListItem = (params: MenuItemParams) => {
```

Comment:
Treat as JSX element instead of regular function

Reply:
Good call